src/daggy.js

```diff
@@ -169,6 +171,21 @@
     }
   }
 
+  function makeConstructorFromObject(fields, proto) {
+    return function(obj) {
+      var values = [];
+      for (var idx = 0; idx < fields.length; idx += 1) {
+        var field = fields[idx];
+        if (!Object.prototype.hasOwnProperty.call(obj, field)) {
+          throw new TypeError('Missing field: ' + field);
+        }
+        var value = obj[field];
+        values.push(value);
```

Comment:
It looks as though `value` is only referenced once, so we could use `values.push(obj[field])` here. :)

Reply:
I have a thing for naming things, but there is no need here.